packages/core/src/StateNode.ts

```diff
@@ -336,6 +325,19 @@ export class StateNode<
     );
   }
 
+  public get initial(): StateNode<TContext, any, TEvent>[] {
```

Comment:
Can you please represent this as an `InitialTransitionDefinition` where that is defined as:

```ts
export interface InitialTransitionDefinition<
  TContext,
  TEvent extends EventObject
> extends TransitionDefinition<TContext, TEvent> {
  target: Array<StateNode<TContext, any, TEvent>>;
}
```

Should be a straightforward change and reduce conflicts.